locales/en/messages.json

```diff
@@ -3052,6 +3052,9 @@
     "firmwareFlasherReleaseTarget": {
         "message": "Target:"
     },
+    "firmwareFlasherTargetWikiUrl": {
```

Comment:
Is this the right place to store this? Are we assuming there will be different wikis dependent on language?

Reply:
Well, just not to have an external defines in code but placed outside, but you are right this are not a language issue. No problem to change.